### ISSUE TITLE
Cannot Load Game Versions. Mods tab will be unavailable. Help.

### ISSUE THREAD

**Issue by FenraVR** [Assistant/ModAssistant#300]
I only get this every time I try to run the mod loader. Tried reinstalling 2-3 times, as well as a full clean reinstall of the game and mod loader. This started happening after trying to run the mod loader once, which worked, then I tried to update the mods that needed updates and it became stuck and had a forever load, so I closed and restarted it.
![image](https://user-images.githubusercontent.com/77898214/105612104-c3178a00-5d87-11eb-8e4f-b7e6b8b0f796.png)
Upon clicking on "agree" in the first tab. 
![image](https://user-images.githubusercontent.com/77898214/105612262-e8f15e80-5d88-11eb-9d12-c2aa0e646961.png)
I haven't been able to run mods since before the multiplayer update, and no matter what I've done I haven't been able to get it to work, and have been stuck with vanilla songs only. I don't have any coding knowledge, and the only thing I can do is put this here and hope someone can help.

**Comment by jcmcmurry** [comment]
I'm in the same boat. The mod loader was working (though mods weren't running, and BSIPA wasn't updating), then I tried uninstalling BSIPA to reinstall it, and suddenly I couldn't access the mods tab and it supplied me with the same messages.

**Comment by FenraVR** [comment]
> I'm in the same boat. The mod loader was working (though mods weren't running, and BSIPA wasn't updating), then I tried uninstalling BSIPA to reinstall it, and suddenly I couldn't access the mods tab and it supplied me with the same messages.

Yeah I'm hoping we get a response quick. Its been months since I've had my rainbow road ex+ fix man

**Comment by Noelliphant** [comment]
Pretty sure it's because beatsaber has updated, so the mod assistant also needs updated to work with the new version of beatsaber. uninstall mods until an update comes out, you can still play your songs, just without the extra effects.

**Comment by FenraVR** [comment]
> Pretty sure it's because beatsaber has updated, so the mod assistant also needs updated to work with the new version of beatsaber. uninstall mods until an update comes out, you can still play your songs, just without the extra effects.

Actually its just that. ALL the mod content is unavailable to me, including custom songs

**Comment by altz47** [comment]
I'm having the same problem and I don't know if this has anything to do with it but my game type is wrong on the options page. 

has anyone got a fix for this yet?

**Comment by ReikoTigon** [comment]
Same here

**Comment by zizip0p** [comment]
I have the same problem. It arose about two months ago. Have to wait two minutes(

**Comment by JacobCavenar** [comment]
The wait is more than two minutes considering you have to wait ages for the mods to download after already waiting. Where is the support for this? I've seen issue tickets for this still open from last July, wtf is going on?

**Comment by zizip0p** [comment]
I have this problem disappeared since version 1.1.18

**Comment by SpeedRunnerFuchs-Wulf** [comment]
I have encountered the "Mod Assistant couldn't load game versions. Mods tab will be disabled" error as of 4/22/2021. The assistant instructed me to reinstall it because of the error, but the error persisted. Any help would be most appreciated.

**Comment by megalon** [comment]
The game versions list is served by BeatMods at [https://versions.beatmods.com/versions.json](https://versions.beatmods.com/versions.json)

If that link is unavailable to you for some reason, then Mod Assistant would not be able to load the game versions.

@SpeedRunnerFuchs-Wulf Are you still having this issue?
Is anyone else still having this issue?

**Comment by DSpoons** [comment]
Yeah, I am in version 1.1.30. I got it to go away once by running as admin but have had no luck in 49/50 tries
